UnitySDK/Assets/ML-Agents/Examples/Soccer/Scripts/SoccerBallController.cs

```diff
@@ -1,26 +1,12 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 
 public class SoccerBallController : MonoBehaviour
 {
     [HideInInspector]
     public SoccerFieldArea area;
-    public AgentSoccer lastTouchedBy; //who was the last to touch the ball
```

Comment:
Are you sure this is not used anywhere ?

Reply:
global search in the project shows it's not being used, not even commented out code.

Reply:
@chriselion showed that `grep` reveals that this property is referenced in a couple of prefabs.  I'll check this out and respond here.